src/library/searchquery.h

```diff
@@ -91,6 +93,23 @@ class TextFilterNode : public QueryNode {
     QString m_argument;
 };
 
+class NullTextFilterNode : public QueryNode {
```

Comment:
The naming is inappropriate if we want to filter for both NULL and empty strings.